follower/follower.go

```diff
@@ -17,9 +17,7 @@ const (
 	peekSize = 1024
 )
 
-var (
-	_ = fmt.Print
-)
+var _ = fmt.Print
```

Comment:
If we're going to touch this line, let's just get rid of it (and the "fmt" import) entirely.